frontend/src/Onboarding/TransferableCreditScreen.tsx

```diff
@@ -198,22 +198,22 @@ const TransferableCreditScreen: React.FC = () => {
 
     const createPlanPromise = () => {
       let schedule, courseCounter;
-      if (!!coopCycle) {
-        schedule = generateBlankCompletedCourseSchedule(
-          academicYear,
-          graduationYear,
-          completedCourseSchedule!,
-          major!,
-          coopCycle!,
-          allPlans
-        );
-      } else {
-        schedule = generateBlankCompletedCourseScheduleNoCoopCycle(
-          academicYear,
-          graduationYear,
-          completedCourseSchedule!
-        );
-      }
+      // if (!!coopCycle) {
```

Comment:
Why did we get rid of this condition?

Reply:
This condition was adding an error, so it was just a temporary fix